bin/bootstrax

```diff
@@ -1302,6 +1306,8 @@ def process_run(rd, send_heartbeats=args.production):
     log.info(f"Starting processing of run {rd['number']}")
     if rd is None:
         raise RuntimeError("Pass a valid rundoc, not None!")
+    elif args.production and rd.get('bootstrax', {}).get('state', None) == 'done':
+        raise RuntimeError(f'{rd["number"]} is done already, do not make a mass')
```

Comment:
Should this say `mess` rather than `mass`? Also, given that the dispatcher guarantees the existence of `bootstrax.state`, are the nested `.get()`s necessary?

Reply:
Divine intervention is not appreciated. 

Thanks for spotting it! 

`are the nested .get()s necessary`, looks like not but also doesnt hurt much.